lib/public/Model.js

```diff
@@ -153,6 +160,9 @@ export default class Model extends Observable {
                 this.logs.fetchAllLogs();
                 this.runs.fetchAllRuns();
                 break;
+            case 'lhc-period-overview':
+                this.lhcPeriods.fetchLhcPeriods();
+                break;
```

Comment:
Please use more recent way of doing stuff: `this.lhcPeriods.loadOverview()` and in it load a sub-model

Reply:
applied